File: Equipments.Domain/Entities/Equipment.cs
namespace Equipments.Domain.Entities
{
    public class Equipment : Entity
    {
        public Guid EquipmentModelId { get; private set; }
        public string Name { get; private set; }

        public Equipment(Guid equipmentModelId, string name)
        {
            EquipmentModelId = equipmentModelId;
            Name = name;
        }

        public void EditName(string name) => Name = name;
    }
}

File: equipments/src/main/java/com/br/raf/equipments/entities/Equipment.java
package com.br.raf.equipments.entities;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import java.util.UUID;
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "equipment")
public class Equipment {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private UUID id;
    private String name;

    @Column(name = "equipment_model_id")
    private UUID equipmentModelId;

}
